tests/test__collection_api.py

```diff
@@ -4224,6 +4224,32 @@ def test_aggregate_group_sum(self):
         }]
         self.assertEqual(expect, list(actual))
 
+    @skipIf(not _HAVE_PYMONGO, 'pymongo not installed')
+    def test__aggregate_group_sum_for_decimal(self):
+        collection = self.db.collection
+        collection.drop()
+        collection.insert_one(
+            {
```

Comment:
Please move this opening curly bracket together with the previous line. (Note that in general I try to reduce the indentation if there's only one value inside).

Reply:
It's reasonable.  I have reformed these codes. Thanks.